features/bootstrap/FeatureContext.php

```diff
@@ -394,21 +394,23 @@ private function getExpectedOutput(PyStringNode $expectedText)
      */
     public function itShouldFail($success)
```

Comment:
I would change the name of the function to `itShouldPassOrFail` to make it clear that it manages both cases

Reply:
I would fix that whole part, like so:
```php
    /**
     * Checks whether previously ran command passed or failed.
     *
     * @Then /^it should (fail|pass)$/
     *
     * @param 'pass'|'fail' $success
     */
    public function itShouldPassOrFail(string $success): void
```